activities/NDA/items/interview_age.jsonld

```diff
@@ -42,6 +50,5 @@
     "pav:createdOn": null,
```

Comment:
let's take some of the metadata stuff out for now.

Reply:
I removed only the createdOn key for now. Let me know if others need to be removed too.

Reply:
let's remove all of them for now.

Reply:
ok. is this required: https://github.com/sanuann/schema-standardization/blob/master/activities/PHQ-9/items/phq9_1.jsonld#L31 ? (was suggested by Hauke)

Reply:
for the moment let's remove it and strip it down to the necessary knowledge pieces. we will likely add things at the item and activity level to add ontological descriptions of how these things relate. something like a relatedconcepts.

Reply:
okay